v0/destinations/candu/transform.js

```diff
@@ -14,9 +15,23 @@ const responseBuilder = (body, { Config }) => {
   payload.context = {
     source: "RudderStack"
   };
+  if (payload.userId) {
+    payload.userId = `${payload.userId}`;
+  }
+  const revisedPayload = {};
+  // here we are trying to check if there is any empty object, array or string inside the payload.
```

Comment:
Why are these new checks needed? Does Candu throw an error otherwise?

Reply:
Candu doesn't throw an error but to prevent sending any void object or array doing these checks.

Reply:
In that case, let's not perform these validations.